Marcio/1.1.1.1 ios app (#1798)
cloudflare/cloudflare-docs: products/1.1.1.1/src/content/setting-up-1.1.1.1/iphone.md

1. Click **Save**.

## 1.1.1.1 iOS app

If you want to configure 1.1.1.1 automatically, you can [download Cloudflare's 1.1.1.1: Faster Internet](https://apps.apple.com/us/app/1-1-1-1-faster-internet/id1423538627) free iOS app.

> 1.1.1.1: Faster Internet automatically configures 1.1.1.1 on any network you connect to. Additionally, it allows you to enable the [WARP mode](https://developers.cloudflare.com/warp-client/), which keeps all your HTTP traffic private and secure.

Reviewer: ```suggestion
1.1.1.1: Faster Internet automatically configures 1.1.1.1 on any network you connect to. It also allows you to enable the [WARP mode](https://developers.cloudflare.com/warp-client/), which gives you all the protection from 1.1.1.1 while additionally keeping all your HTTP traffic private and secure.
```
^ this may be a bit more clarifying, wdyt?

Author: I've added this and additional info

To set up 1.1.1.1: Faster Internet:

1. Download [1.1.1.1: Faster Internet from the app store](https://apps.apple.com/us/app/1-1-1-1-faster-internet/id1423538627).

> 1. Launch 1.1.1.1: Faster Internet and accept its Terms of Service.

Reviewer 2: ```suggestion
1. Launch 1.1.1.1: Faster Internet and accept the Terms of Service.
```

1. Install the VPN profile that allows your phone to connect securely to 1.1.1.1.

> You are now using 1.1.1.1 for your Internet queries. If you want to add an extra layer of security with HTTPS encryption, toggle the WARP button.

Reviewer 2: ```suggestion
You are now using 1.1.1.1 for your Internet queries. If you want to add an extra layer of security with HTTPS encryption, toggle the WARP button to be enabled.
```

1. Click **Save**.

## 1.1.1.1 iOS app

If you want to configure 1.1.1.1 automatically, you can [download Cloudflare's 1.1.1.1: Faster Internet](https://apps.apple.com/us/app/1-1-1-1-faster-internet/id1423538627) free iOS app.

> 1.1.1.1: Faster Internet automatically configures 1.1.1.1 on any network you connect to. It also allows you to encrypt your DNS queries to 1.1.1.1 DNS resolver or enable the [WARP mode](https://developers.cloudflare.com/warp-client/), which gives you all the protection from 1.1.1.1 while additionally keeping all your HTTP traffic private and secure. By default, 1.1.1.1: Faster Internet is setup with the WARP mode.

Reviewer 3: ```suggestion
1.1.1.1: Faster Internet automatically configures 1.1.1.1 on any network you connect to. It also allows you to encrypt your DNS queries to the 1.1.1.1 DNS resolver or enable [WARP mode](https://developers.cloudflare.com/warp-client/), which gives you all the protection from 1.1.1.1 while additionally keeping all your HTTP traffic private and secure. By default, 1.1.1.1: Faster Internet is set up with WARP mode.
```

To set up 1.1.1.1: Faster Internet:

1. Download [1.1.1.1: Faster Internet from the app store](https://apps.apple.com/us/app/1-1-1-1-faster-internet/id1423538627).
1. Launch 1.1.1.1: Faster Internet and accept the Terms of Service.
1. Install the VPN profile that allows your phone to connect securely to 1.1.1.1.

> You are now using 1.1.1.1 for your Internet queries. If you want to add an extra layer of security and encrypt all of your traffic with HTTPS, toggle the WARP button to *Connected*.

Reviewer 3: ```suggestion
You are now using 1.1.1.1 for your Internet queries. If you want to add an extra layer of security and encrypt all of your traffic with HTTPS, toggle the **WARP** button to *Connected*.
```

> You are now using 1.1.1.1 for your Internet queries. If you want to add an extra layer of security and encrypt all of your traffic with HTTPS, toggle the WARP button to *Connected*.

Reviewer 3: It's not entirely clear what we mean by "Internet queries" — is it only DNS queries?

Also, above we say that the default is WARP mode, and yet we're turning it on here (maybe I'm missing something).

Author: Changed wording in new commit to improve clarity.
